Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: docat
+description: A Helm chart for Kubernetes
```

Comment:
should that contain a description what this heml chart does?

Reply:
I updated the Pull Request description :) This was generated automatically, it's to be replaced, but I can update on this pull request if you want :)